pkg/csi/cinder/openstack/openstack.go

```diff
@@ -148,7 +149,14 @@ func InitOpenStackProvider(cfgFiles []string, httpEndpoint string) {
 			klog.Infof("metrics available in %q", httpEndpoint)
 		}()
 	}
-
+	if len(cfgFiles) == 0 {
+		klog.Infof("Using internal cloud config file: %s", cfgFiles)
+		if err := createCfgFile(configFiles[0], regionName); err != nil {
+			klog.Fatalf("Failed to create cloud config file: %v", err)
+		}
+		return
+	}
+	klog.Infof("Using provided cloud config files: %s", cfgFiles)
```

Comment:
There's no need to separate Claymore from non-Claymore as long as we provide different secrets (dummy and real) to both. In my DaemonSet PR, I removed the complicated step of building `cloud.conf` from the cloud provider for this reason. I suggest we don’t add the `USE_CLAYMORE` flag.

Reply:
https://platform9.atlassian.net/browse/NGPC-1057